source/winUser.py

```diff
@@ -514,6 +514,10 @@ def FindWindow(className, windowName):
 IDCANCEL=3
 
 def MessageBox(hwnd, text, caption, type):
+	if hasattr(text, 'decode'):

```

Comment:
Please use `isinstance(text, bytes)` instead. Otherwise, this will lead to unnecessary decoding on python 2 unicode strings.